### PR TITLE
fix(core): remove grid list component css from Ngx

### PR DESCRIPTION
**Pull request by DeepakSap14**
#### Please provide a link to the associated issue.
Part of #5025 

#### Please provide a brief summary of this pull request.
This PR removes the styles present in Ngx. Those styles were already moved to styles repo with [PR](https://github.com/SAP/fundamental-styles/pull/2525) 

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [NA] update `README.md`
- [NA] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by DeepakSap14** [comment]
can be reviewed once PR #6396 is merged.

File: libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// these css are related to drag and drop
 @mixin fd-rtl {
```
Comment (N1XUS) [inline libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.scss:2]: Are those mixins used anywhere?

Comment (DeepakSap14) [reply libs/core/src/lib/grid-list/components/grid-list-item/grid-list-item.component.scss]: removed. not getting used anymore.

**Review by platon-rov — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
Screen Shot 2021-09-03 at 11 41 11 AM

Why are the action button in the middle of nowhere?

**Comment by DeepakSap14** [comment]
> Screen Shot 2021-09-03 at 11 41 11 AM
> 
> Why are the action button in the middle of nowhere?

Issue is coming from `toolbar component` because of recent changes in that component. will check and fix that.

**Review by Betrozov — APPROVED** [review]

**Comment by DeepakSap14** [comment]
> Screen Shot 2021-09-03 at 11 41 11 AM
> 
> Why are the action button in the middle of nowhere?

@InnaAtanasova 
Toolbar bug is fixed now.
it is also fixing grid list bug pointed by you.
please check now.

**Comment by puru-hk** [comment]
Looks fine

**Review by puru-hk — APPROVED** [review]

File: libs/core/src/lib/grid-list/components/grid-list-filter-bar/grid-list-filter-bar.component.scss [hunk 2]
```diff
@@ -1,3 +1 @@
-.fd-grid-list__filter-button:active:hover {
-    color: #fff !important;
-}
+
```
Comment (mikerodonnell89) [inline libs/core/src/lib/grid-list/components/grid-list-filter-bar/grid-list-filter-bar.component.scss:1]: this file can be deleted

**Review by mikerodonnell89 — APPROVED** [review]
Looks good please just remove that empty SCSS file before merge

File: libs/core/src/lib/grid-list/components/grid-list-filter-bar/grid-list-filter-bar.component.scss (hunk 2 above)
Comment (DeepakSap14) [reply libs/core/src/lib/grid-list/components/grid-list-filter-bar/grid-list-filter-bar.component.scss]: removed scss file and its reference.

**Pull request merged**